### DISCUSSION TITLE
Welcome to Calculator Discussions!

### DISCUSSION THREAD

**Discussion by ApplicationCoding** [ApplicationCoding/calculator#24] [Announcements]
## 👋 Welcome!
  We’re using Discussions to talk about ideas, requests, and questions. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by YojanProductions** [comment]
Can u guys add an updated UI? , thanks

**Reply by ApplicationCoding** [reply]
@YojanProductions, you're issue https://github.com/ApplicationCoding/calculator/issues/11 is the same as you're comment, right? Or you have new suggestions?

**Reply by YojanProductions** [reply]
O , my suggestion is a little bit different my suggestion is the UI looking like a a real calculator

**Reply by ApplicationCoding** [reply]
Ohh, yes. That will be on Calculator v2.0. But, the development is temporarily paused for now.
